src/ec/packages/ec-component-dropdown-legacy/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@ecl-twig/ec-component-dropdown-legacy",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.9.1",
```

Comment:
it should be version 2.11.0 (same for dependencies)

Reply:
done